### PR TITLE
WINC-858: Prep for 7.0.0 release

### PR DESCRIPTION
**Pull request by jrvaldes**
This change bumps the operator and the OCP version
according to WMCO 7.0.0 release.

Ran:
```
  make bundle
```

### CODE REVIEW

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

Thanks for doing this Jose

**Review by sebsoto — APPROVED** [review]

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=5.0.0 <6.0.0'
+    olm.skipRange: '>=6.0.0 <7.0.0'
```
Comment (aravindhp) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: @mansikulkarni96 @saifshaikh48 I wonder if we can get `render-templates` to update all places where version is being used. Is that part of release improvement epic?

**Comment by jrvaldes** [comment]
/hold 

To follow up on Azure CI [issue](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/1237/pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator/1569493001061797888)

**Comment by aravindhp** [comment]
/retitle Prep for 7.0.0 release

Lets be consistent with our naming. @jrvaldes does this also need to be associated with WINC-858?

**Comment by jrvaldes** [comment]
> /retitle Prep for 7.0.0 release
> 
> Lets be consistent with our naming. @jrvaldes does this also need to be associated with WINC-858?

It doesn't hurt. Done.

**Pull request merged**